blog/news.json

```diff
@@ -1 +1 @@
-[{"title": "July 2024 Releases", "slug": "2024-07-23-july-releases", "authors": ["beeankha"], "tags": ["announcement", "conda", "conda-build", "conda-libmamba-solver"], "description": "Conda 24.7.0/.1, conda-build 24.7.0, and conda-libmamba-solver 24.7.0 have been released! \ud83c\udf89\n", "image": "img/blog/2024-07-23-july-releases/banner.png", "date": "2024-07-23T00:00:00"}, {"title": "June 2024 Releases", "slug": "2024-06-14-june-releases", "authors": ["dashagurova"], "tags": ["announcement", "conda-index", "conda-package-handling", "conda-package-streaming"], "description": "Conda-index 0.5.0, conda-package-handling 2.3.0 and conda-package-streaming 0.10.0 have been released! \ud83c\udf89\n", "image": "img/blog/2024-06-14-june-releases/banner.png", "date": "2024-06-14T00:00:00"}, {"title": "Developer Guide: A Tale of Five Prefixes", "slug": "2024-05-20-prefixes", "authors": ["kenodegard", "chatgpt"], "tags": ["conda", "context", "prefix", "prefixes", "api"], "description": "Prefixes, prefixes, prefixes! All the prefixes!", "image": "img/blog/2024-05-20-prefixes/banner.png", "date": "2024-05-20T00:00:00"}, {"title": "April and May 2024 Releases", "slug": "2024-05-13-april-and-may-releases", "authors": ["beeankha"], "tags": ["announcement", "conda", "conda-build"], "description": "Conda 24.4.0, conda 24.5.0, and conda-build 24.5.0 have been released! \ud83c\udf89\n", "image": "img/blog/2024-05-13-april-and-may-releases/banner.png", "date": "2024-05-13T00:00:00"}, {"title": "March 2024 Releases", "slug": "2024-03-20-march-releases", "authors": ["beeankha"], "tags": ["announcement", "conda", "conda-build"], "description": "Conda 24.3.0 and conda-build 24.3.0 have been released! \ud83c\udf89\n", "image": "img/blog/2024-03-20-march-releases/banner.png", "date": "2024-03-20T00:00:00"}]
```

Comment:
Unrelated to this PR, it would be nice if this file would be pretty-printed for a better diff.

Reply:
Agree, I've opened https://github.com/conda/conda-dot-org/pull/203